task2-3/createAndFillDB.sql

```diff
@@ -8,6 +10,20 @@ CREATE TABLE users (
 	is_deleted boolean
 );
 
+CREATE TABLE groups (
+	id text PRIMARY KEY,
+	name text,
+	permissions text []
```

Comment:
Wow, yes. It will also save size on the disk. Because **text** type reserving more place. Better to use **tinytext** which is equal **varchar(255)** as I understood